src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -110,6 +112,33 @@ const SubmissionDetails = _.flow(
   )(workflows)
 
   const { succeeded, failed, running, submitted } = _.groupBy(wf => collapseStatus(wf.status), workflows)
+
+  // Note: This 'archiveLimitYears' value should reflect the current 'archive-delay' value configured for PROD in firecloud-develop's
+  // 'cromwell.conf.ctmpl' file:
+  const archiveLimitYears = 6
+  const archiveLimitString = `${archiveLimitYears} year${archiveLimitYears > 1 ? 's' : ''}`
+  const isArchived = statusLastChangedDate => differenceInDays(Date.parse(statusLastChangedDate), Date.now()) > (archiveLimitYears * 365.24)
+
+  const archivedInfoIcon = ({ name, iconOverride }) => {
+    return h(InfoBox, {
+      style: { color: colors.secondary(), margin: '0.5rem' },
+      tooltip: `${name} unavailable. Click to learn more.`,
+      iconOverride
+    }, [
+      div({ style: Style.elements.sectionHeader }, 'Workflow Details Archived'),
+      div({ style: { padding: '0.5rem 0' } }, [`This workflow's details have been archived (> ${archiveLimitString} old).`]),
+      div([
+        'Please refer to the ',
+        h(Link, {
+          href: 'https://support.terra.bio/hc/en-us/articles/360060601631',
+          ...Utils.newTabLinkProps
+        }, ['Workflow Details Archived ', icon('pop-out', { size: 18 })]),
+        ' support article for more details.'
```

Comment:
Nit: include the words "support article" in the linked text
```suggestion
        'Please refer to the ',
        h(Link, {
          href: 'https://support.terra.bio/hc/en-us/articles/360060601631',
          ...Utils.newTabLinkProps
        }, ['Workflow Details Archived  support article', icon('pop-out', { size: 18 })]),
        ' for more details.'
```